services/employee.js

```diff
@@ -0,0 +1,81 @@
+const employeeRepository = require('../repositories/employee');
+const errors = require("../configuration/errors");
+
+module.exports = {
+  async createEmployee(employeeData) {
+    try {
+      return await employeeRepository.createEmployee(employeeData);
+    } catch (error) {
+      throw new Error(errors.FAILED_TO_CREATE_EMPLOYEE);
+    }
+  },
+
+  async getEmployeeById(employeeId) {
+    try {
+      const employee = await employeeRepository.getEmployeeById(employeeId);
+      if (!employee) {
+        throw new Error(errors.EMPLOYEE_NOT_FOUND);
+      }
+      return employee;
+    } catch (error) {
+      throw new Error(errors.FAILED_TO_GET_EMPLOYEE);
+    }
+  },
+
+  async getAllEmployees() {
+    try {
+      const employees = await employeeRepository.getAllEmployees();
+      const simplifiedEmployees = employees.map((employee) => {
```

Comment:
this is not how we do this. Please use orm and do it in repository. This approach is same as before. We need to do this on db call and not here. https://stackoverflow.com/questions/67809187/mongoose-how-to-return-only-specific-field

Reply:
fixed